### PR TITLE
fix broken links in upgrade/update node sections

### PR DESCRIPTION
**Pull request by meganskye**
address poor quality AI indexing in the context of node updates, and implement standardized edits.

### CODE REVIEW

File: docs/nodes/archive-node/docker.md [hunk 1]
```diff
@@ -150,15 +149,15 @@ eg.
...
-### Indexers and oracles
+### Indexers and Oracles
 
-To access data from indexers (lke The Graph) or Oracles (like Chainlink), you need to add the debug flags below to the node launch command, after the `astar-collator` line:
+To access data from indexers (like The Graph) or Oracles (like Chainlink), add the follwing debug flags to the node launch command, after the `astar-collator` line:
 
 `--ethapi=debug`
 
-### Upgrade node
+## Upgrade node / Update node
```
Comment (midegdugarova) [inline docs/nodes/archive-node/docker.md:158]: is it not either or?

File: docs/nodes/node-maintenance.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ sidebar_position: 5
 
 ## Backup
 
-Maintaining a backup node always sync for a collator is vital to make sure you always produce blocks and avoid being slashed. We highly recommend to have it on a different location and hosted on a different provider.
+Maintaining a backup node that is in sync with a collator is vital to ensuring continuous and interrupted block production, and to avoid the possibility of being slashed. We highly recommend the backup node be hosted in a different physical location, with a different provider.
```
Comment (midegdugarova) [inline docs/nodes/node-maintenance.md:9]: did you mean uninterrupted?

**Review by midegdugarova — APPROVED** [review]

File: docs/nodes/archive-node/docker.md (hunk 1 above)
Comment (meganskye) [reply docs/nodes/archive-node/docker.md:158]: whichever one is more appropriate, as long as both 'update node' and 'upgrade node' are present in the blurb for the AI to notice.

File: docs/nodes/node-maintenance.md (hunk 2 above)
Comment (meganskye) [reply docs/nodes/node-maintenance.md:9]: thank you! i missed that :)

**Pull request merged**